### PR TITLE
Java path doc update for Mac OS

### PR DESCRIPTION
**Pull request by lcampos**
### What does this PR do?
Adds a note on how to get the Java path for Mac OS. Original PR #2899.

### What issues does this PR fix or reference?
@W-8836836@

### CODE REVIEW

File: docs/_articles/en/getting-started/java-setup.md [hunk 1]
```diff
@@ -27,6 +27,8 @@ Inside Visual Studio Code, you will need to set the `salesforcedx-vscode-apex.ja
 
 MacOS:
 
+For Mac OS X 10.5 or later, run `/usr/libexec/java_home` in your terminal to get the default JDK location. To find all installed JDKs, use `/usr/libexec/java_home -V`.
```
Comment (ekapner) [inline docs/_articles/en/getting-started/java-setup.md:30]:
```suggestion
For Mac OS X 10.5 or later, run `/usr/libexec/java_home` in your Terminal to get the default JDK location. To find all installed JDKs, use `/usr/libexec/java_home -V`.
```

Comment (ekapner) [reply docs/_articles/en/getting-started/java-setup.md:30]: I did a quick peek, and I know we aren't totally consistent. So I found some Apple dev docs online, and they capitalize "Terminal" so we'll just fix them as we run across it.

File: docs/_articles/en/getting-started/java-setup.md [hunk 2]
```diff
@@ -94,6 +98,8 @@ Inside Visual Studio Code, you will need to set the `salesforcedx-vscode-apex.ja
...
```
Comment (ekapner) [inline docs/_articles/en/getting-started/java-setup.md:101]:
```suggestion
For Mac OS X 10.5 or later, run `/usr/libexec/java_home` in your Terminal to get the default JDK location. To find all installed JDKs, use `/usr/libexec/java_home -V`.
```

File: docs/_articles/en/getting-started/java-setup.md [hunk 3]
```diff
@@ -65,6 +67,8 @@ Inside Visual Studio Code, you will need to set the `salesforcedx-vscode-apex.ja
...
```
Comment (ekapner) [inline docs/_articles/en/getting-started/java-setup.md:70]:
```suggestion
For Mac OS X 10.5 or later, run `/usr/libexec/java_home` in your Terminal to get the default JDK location. To find all installed JDKs, use `/usr/libexec/java_home -V`.
```

File: docs/_articles/en/getting-started/java-setup.md (hunk 1 above)
Comment (lcampos) [reply docs/_articles/en/getting-started/java-setup.md:30]: Sounds good, I'll make the change

File: docs/_articles/en/getting-started/java-setup.md [hunk 4]
```diff
@@ -114,6 +120,8 @@ If you are still running Java 8 the values for `salesforcedx-vscode-apex.java.ho
...
```
Comment (ekapner) [inline docs/_articles/en/getting-started/java-setup.md:123]:
```suggestion
For Mac OS X 10.5 or later, run `/usr/libexec/java_home` in your Terminal to get the default JDK location. To find all installed JDKs, use `/usr/libexec/java_home -V`.
```

**Review by ekapner — COMMENTED** [review]
capitalize Terminal

**Pull request merged**